addon/components/impagination-dataset.js

```diff
@@ -46,6 +47,18 @@ export default Ember.Component.extend({
 
     this.setInitialState();
     this.get('dataset').setReadOffset(this.get('read-offset') || 0);
+  },
+
+  actions: {
+    reset: () => {
```

Comment:
Woops, this was not supposed to make it in there. Yet another reason for testing all new feature APIs.